### PR TITLE
Adds two new whiteships, the "Personal Transit Shuttle" and the "Obelisk"

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

Adds two new shuttles, the Personal Transit Shuttle, and the Obelisk
Also fixes up some grammar issues in test_whiteship_sizes()
### _Personal Transit Shuttle_
A small ship for station-to-station travel, now left to ruin because of an unwise looting decision.
Has bolt controls for exterior airlocks, and a sleeping quarter(s?) for sleeping
SMES and a generator to make sure power is a thing
Has three shambling corpses for enemies,
Loot: 1 random medkit, 1 random amount of gauze, a flashlight, some materials, a tool, a plushie, some food
2022-11-15 07_18_20-Window

### _The Obelisk_
A relatively big ship for long-term exploration, its crew massacred due to questionable religious choices.
Has bolt controls for exterior airlocks, a bedroom for sleeping (bunk beds when)
SMES and a generator for power
Plenty proteons and a juggernaut to guard the bridge.
Contains 3 medkits, a random amount of gauze, a crate with some loot, some cash, a cigar, and some janitorial supplies
2022-11-05 10_44_16-Window

## Why It's Good For The Game

Whiteship variety is always nice i think?

## Changelog

:cl:
add: Added two new whiteships, the "Personal Transit Shuttle" and the "Obelisk"
/:cl:

### CODE REVIEW

File: _maps/shuttles/whiteship_personalshuttle.dmm [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+	zombiejob = "Assistant"
+	},
+/turf/open/floor/plating,
+/area/shuttle/abandoned/engine)
+"by" = (
+/obj/machinery/suit_storage_unit/standard_unit{
+	desc = "An industrial suit storage device carrying retro space suits. Neat!";
+	helmet_type = /obj/item/clothing/head/helmet/space;
+	suit_type = /obj/item/clothing/suit/space
+	},
```
Comment (Fikou) [inline _maps/shuttles/whiteship_personalshuttle.dmm:89]: this object should be defined in the codebase

Comment (mc-oofert) [reply _maps/shuttles/whiteship_personalshuttle.dmm:89]: i literally copied this from another whiteship but sure

**Comment by mc-oofert** [comment]
apparently the rerequest review button removes other reviewers

**Comment by mc-oofert** [comment]
> also make sure personal transit has bolt buttons

im sure it does, image might not be up to date

**Review by Maurukas — CHANGES_REQUESTED** [review]
Everything looks good. Please merge the latest master to pull in the 515 changes.

**Comment by mc-oofert** [comment]
> Everything looks good. Please merge the latest master to pull in the 515 changes.

done

**Pull request merged**